announcements/2020-Week-37/apps.md

```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Releases (September 10, 2020):zap:
```

Comment:
```suggestion
# :zap: This Week in Provider-Independent Announcements (September 10, 2020):zap:
```

Since this will include a CVE from @stone-z I think "Announcements" is more accurate than "Releases"